tests/phpunit/tests/class-wp-templates-registry-test.php

```diff
@@ -0,0 +1,192 @@
+<?php
+/**
+ * Test WP_Block_Templates_Registry class.
+ *
+ * @covers WP_Block_Templates_Registry
+ */
+class WP_Block_Templates_Registry_Test extends WP_UnitTestCase {
```

Comment:
Rename and relocate file to `block-templates/WpBlockTemplatesRegistry.php` too

```suggestion
class Tests_Block_Templates_WpBlockTemplatesRegistry extends WP_UnitTestCase {
```

Reply:
Done in fa9e6ad681636255c2ee3f48eafc3aa02f84b921.